packages/pglite/src/pglite.ts

```diff
@@ -419,6 +419,8 @@ export class PGlite
             'INITDB created a new datadir, but an alternative db/user was requested',
           )
         }
+        // time to mark extensions that need special treatment on startup.
+        if (options.postgresqlConf) this.updateConf(options.postgresqlConf)
```

Comment:
If I understand correctly, this runs after initdb. I think I need to add some tests to check is PG correctly reads and process it.

Reply:
it runs only when initdb has created a db. If used for changing config later it should be called manually and so documented somewhere.